src/user/serializers.py

```diff
@@ -700,7 +700,12 @@ def get_item(self, action):
         if type(item) in ignored_items:
             return None
 
-        if isinstance(item, Paper):
+        if isinstance(item, Withdrawal):
+            from reputation.serializers import WithdrawalSerializer
+            serializer = WithdrawalSerializer
+            context = {}
+            _context_fields = {}
```

Comment:
Let's make this into a dynamic serializer to keep things consistent.

Reply:
I'm not sure how to do that

Reply:
In `reputation/serializers.py` add
```
class DynamicWithdrawalSerializer(DynamicModelFieldSerializer):
    user = serializers.SerializerMethodField()

    class Meta:
        model = Withdrawal
        fields = '__all__'

    def get_user(self, withdrawal):
        context = self.context
        _context_fields = context.get('rep_dws_get_user', {})
        serializer = DynamicUserSerializer(
            withdrawal.user,
            context=context,
            **_context_fields
        )
        return serializer.data
```

then

```suggestion
            from reputation.serializers import DynamicWithdrawalSerializer
            serializer = DynamicWithdrawalSerializer
```

Reply:
I'll update this  & test it in a tech debt sprint